_sass/main.scss

```diff
@@ -1,6 +1,8 @@
 
 @import './header.scss';
 
+$small: 600px;
```

Comment:
can we follow the naming convention in .header/scss for the screen size, and if possible import it?